README.md

```diff
@@ -54,19 +54,28 @@ Then to run the tests use ```npm test```
 
 On Windows, use ```npm run testwindows```
 
-# Caveats
+## Caveats
 
-## Two sync loops per sync frequency
+### Two sync loops per sync frequency
 Two sync loops may be invoked per sync frequency if the server-side sync frequency
 differs from the client-side frequency.
 
 This is because the client and server sync frequencies are set independently.
 Setting a long frequency on a client does not change the sync frequency on the
 server.
 
-The `syncFrequency` value of the dataset on the server should be set to the 
+The `syncFrequency` value of the dataset on the server should be set to the
 `sync_frequency` value of the corresponding dataset on the client to avoid this.
 
 For example:
   * `sync_frequency` on the client-side dataset is also set to 120 seconds.
   * `syncFrequency` on the server-side dataset is set to 120 seconds.
+
+## Sync API logging
```

Comment:
@grdryn @wei-lee I like this idea of having a more standard approach to logging. Would you consider having a standard logger across the entire fh-mbaas-api? It seems sync does not share the common logger found [here](https://github.com/feedhenry/fh-mbaas-api/blob/master/lib/consolelogger.js). If it did then enabling/disabling SDK logs would be easy to approach.

I find the `debug` module is a nice to use logger in libraries since it can easily be disabled and enabled like so:

```
export DEBUG=name-of-logger
```

So for the mbaas-api we could do things like:

```
# just enable sync and service verbose logs
export DEBUG=fh-mbaas-api:sync,fh-mbaas-api:service

# enable all fh-mbaas-api logs
export DEBUG=*
```

Then in code you can have:

```
// sync.js

// you could wrap debug to add custom log formatting...
var log = require('debug')('fh-mbaas-api:sync');
```

For some apps built by consulting we use [fh-bunyan](https://www.npmjs.com/package/fh-bunyan) too since it auto configures itself based on the `FH_ENV`. When running locally (FH_USE_LOCAL_DB is set) it will use `trace` level. It can also be configured by setting an `BUNYAN_FH_LEVEL` env var.

Just some of my 2 cents. Nice work on the recent sync stuff! 👍